zoom_monitor.py

```diff
@@ -87,19 +87,16 @@ def _join_meeting(self):
         self._wait_for_element(By.XPATH, PARTICIPANTS_BTN, timeout_s=30)
         self._logger.info("logged into Zoom successfully")
 
-    def _wait_for_element(self, approach, value, timeout_s=5):  # Helper function
+    def _wait_for_element(self, approach, value, timeout_s=5):  # Helper fn
```

Comment:
Trying to stay under 80 characters per line, and I didn't want to break this up onto multiple lines. :upside_down_face: I'm open to other alternatives (including tolerating long lines), if you prefer something else.

Reply:
2¢ because I love a good bikeshed 😄 - I agree that abbreviations like this should be avoided; it's nice where possible to avoid making assumptions about folks' experience with english and familiarity with certain abbreviations.

This function has a docstring, do we need both a docstring and comment saying "helper function"? Sort of feels like the comment is meaningless

Reply:
TBH, I think the comment mentioning that it's a helper function is useless, and I was trying to keep it in because apparently someone else thought it was important. I like Michael's suggestion; I'll just take it out! Hopefully the name of the function is already more useful than marking it as a helper.